app/assets/stylesheets/beta.css.scss

```diff
@@ -0,0 +1,42 @@
+@import "compass";
+@import "colors";
+
+.beta-signup {
+  margin-top: 30%;
+  width: 100%;
+
+  .fi-laptop {
+    font-size: 1.8em;
+    margin-right: 3px;
+    vertical-align: middle;
+  }
+}
+
+#beta_signin {
```

Comment:
Avoid using id selectors<br>Selector `beta_signin` should be written in lowercase with hyphens